jupyter/internal/mimesniff/sniff.go

```diff
@@ -0,0 +1,269 @@
+// Copyright 2017 The Neugram Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// the following has been extracted from the Go stdlib at revision:
```

Comment:
Why are you extracting this from the go stdlib? You could use http.DetectContentType directly.

Reply:
I was worried about importing the whole 'http' package for that one function.
I'll just import it.

Reply:
done.